### DISCUSSION TITLE
subscribe to an item

### DISCUSSION THREAD

**Discussion by xu4wang** [directus/directus#12139] [Archive]
Is there a way to subscribe to an item and receive email notification when the item had been changed?

**Comment by rijkvanzanten** [comment]
No, not yet. That being said, we're working on support for this as we speak, so it will be a thing in the near future 👍🏻

**Comment by Thermal-Shock** [comment]
I'm new to directus but if you able this seems like it could be done with a custom API hook. This would also give you the ability to filter your item/collection and have a corresponding collection/table from where to pull the email template including placeholders. 

As @rijkvanzanten mentioned, this may be mute if it is planned to be available as standard by way of webhooks or some other feature. It would be nice if webhooks could also have filters so they are only triggered for specific data. 

All the best.

**Reply by rijkvanzanten** [reply]
That's also true! You could always use extensions to implement anything that's missing for your particular use case 🙂

**Comment by xu4wang** [comment]
@Thermal-Shock thanks for the answer. Will try.

@rijkvanzanten thanks for the good news.  I saw a lot of features on your plate. ;-) I will keep monitoring.
